pkg/scheduler/framework/plugins/podtopologyspread/filtering.go

```diff
@@ -347,20 +347,21 @@ func (pl *PodTopologySpread) Filter(ctx context.Context, cycleState *framework.C
 		return nil
 	}
 
+	logger := klog.FromContext(ctx)
 	podLabelSet := labels.Set(pod.Labels)
 	for _, c := range s.Constraints {
 		tpKey := c.TopologyKey
 		tpVal, ok := node.Labels[c.TopologyKey]
 		if !ok {
-			klog.V(5).InfoS("Node doesn't have required label", "node", klog.KObj(node), "label", tpKey)
+			logger.V(4).Info("Node doesn't have required label", "node", klog.KObj(node), "label", tpKey)
```

Comment:
Why V(4) here.

Reply:
Sorry, there may be some problems with batch change, I will change back to V(5).